lib/kino/explorer.ex

```diff
@@ -56,7 +56,11 @@ defmodule Kino.Explorer do
         }
       end
 
-    info = %{name: name, features: [:pagination, :sorting]}
+    info = %{
+      name: name,
+      features: [:export, :pagination, :sorting],
+      export: %{formats: [:csv, :parquet]}
```

Comment:
Let's also add support for NDJSON:

```suggestion
      export: %{formats: ["CSV", "NDJSON", "Parquet"]}
```

Reply:
Oh, I see that the atom is what is used to download, but lets keep it as strings, because we should respect casing?